docs/manual/endpoints.rst

```diff
@@ -0,0 +1,282 @@
+      Authorization: Basic YWRtaW46YWRtaW4=
+      Content-Type: application/json
+      password: YWRtaW46YWRtaW4=
+
+      {
+        "email" : {
+          "address" : "newsampleuser@sanctionco.com",
+          "verified" : false,
+          "verificationToken" : null
+        },
+        "password" : "12345",
+        "properties" : {
+          "myCustomProperty" : "My properties have changed"
+        }
+      }
+
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+        "email" : {
+          "address" : "newsampleuser@sanctionco.com",
+          "verified" : false,
+          "verificationToken" : null
+        },
+        "password" : "12345",
+        "properties" : {
+          "myCustomProperty" : "My properties have changed"
+        }
+      }
+
+   :query email: the existing email address of the user to update. This is optional, and only
+         required if the email is to be changed.
+   :reqheader Authorization: basic authentication application name and secret
+   :reqheader password: the (hashed) password of the user to update
+   :statuscode 200: user was successfully updated
+
+Get User
+========
+
+.. http:get:: /users
+
+   Updates an existing user in the database.
+
+   **Example**:
+
+   .. http:example:: curl wget httpie
+
+      GET /users?email=sampleuser@sanctionco.com HTTP/1.1
+      Authorization: Basic YWRtaW46YWRtaW4=
+      Content-Type: application/json
+      password: YWRtaW46YWRtaW4=
+
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+        "email" : {
+          "address" : "sampleuser@sanctionco.com",
+          "verified" : false,
+          "verificationToken" : null
+        },
+        "password" : "12345",
+        "properties" : {
+          "myCustomProperty" : "Hello World"
+        }
+      }
+
+   :query email: the email address of the user
+   :reqheader Authorization: basic authentication application name and secret
+   :reqheader password: the (hashed) password of the user
+   :statuscode 200: the operation was successful
+
+Delete User
+===========
+
+.. http:delete:: /users
+
+   Deletes a user from the database.
+
+   **Example**:
+
+   .. http:example:: curl wget httpie
+
+      DELETE /users?email=sampleuser@sanctionco.com HTTP/1.1
+      Authorization: Basic YWRtaW46YWRtaW4=
+      Content-Type: application/json
+      password: YWRtaW46YWRtaW4=
+
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+        "email" : {
+          "address" : "sampleuser@sanctionco.com",
+          "verified" : false,
+          "verificationToken" : null
+        },
+        "password" : "12345",
+        "properties" : {
+          "myCustomProperty" : "Hello World"
+        }
+      }
+
+   :query email: the email address of the user
+   :reqheader Authorization: basic authentication application name and secret
+   :reqheader password: the (hashed) password of the user
+   :statuscode 200: the operation was successful
```

Comment:
I noticed on all these endpoints there you only include the successful response codes. Can you also include all the error response code the endpoints could return. I know it can help if someone were to ever create some sort of client library in a language we haven't implemented.